### DISCUSSION TITLE
Where list of peers is stored in an Android devices?

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#8830] [Q&A]
I would like to save my PC's .tomls file (peers) in my Android Device to be able to see them also in my smartphone.
Where RustDesk stores the list of peers in an Android device?
Thank you
Aria

**Comment by basilgello** [comment]
`/data/user/X/com.carriez.flutter_hbb/app_flutter`

BTW @rustdesk it would be really great to allow overriding peer list and custom options from `/sdcard/RustDesk` on Android. That'd help provision devices with MDMs and, for free users, have consistent config backups

**Reply by ghost** [reply]
I haven't such folder in my Samsung Galaxy Note 20 Ultra.
There is \Android\data\com.carriez.flutter_hbb\files but the folder is empty.

**Reply by basilgello** [reply]
It is on internal storage and you can access it only via root, not even ADB. What you _can_ try is to `adb backup`, edit the backup and restore it with `adb restore`.
